ui/hooks/bridge/useBridging.ts

```diff
@@ -43,6 +43,7 @@ const useBridging = () => {
   const isMarketingEnabled = useSelector(getDataCollectionForMarketing);
   const providerConfig = useSelector(getProviderConfig);
   const keyring = useSelector(getCurrentKeyring);
+  // @ts-expect-error keyring type is wrong maybe?
```

Comment:
Is this exception expected as part of this refactoring effort?

Reply:
yes, the existing types used in various places around our codebase don't match what the code actually represents, so we get errors like this when porting from JS to TS. If someone fixes the types this line will _cause_ a compilation error, which is good as it will be the comment should just be removed at that time.